nullaway/src/main/java/com/uber/nullaway/handlers/OptionalEmptinessHandler.java

```diff
@@ -49,9 +51,12 @@
  */
 public class OptionalEmptinessHandler extends BaseNoOpHandler {
 
-  private static String OPTIONAL_PATH = "java.util.Optional";
+  @Nullable private Set<Optional<Type>> optionalTypes;
```

Comment:
Why is this a `Set<Optional<Type>>` rather than a `Set<Type>`? We could just filter out any type paths for which `state.getTypeFromString(type)` returns `null`.

Also, it probably should be an `ImmutableSet`, since it won't change after the call to `onMatchTopLevelClass`

Reply:
Done 👍